parlai/scripts/train_model.py

```diff
@@ -242,6 +241,45 @@ def load_eval_worlds(agent, opt, datatype):
     return worlds
 
 
+def _run_single_eval(opt, valid_world, max_exs):
+    # run evaluation on a single world
+    valid_world.reset()
+
+    cnt = 0
+    max_cnt = max_exs if max_exs > 0 else float('inf')
+    while not valid_world.epoch_done() and cnt < max_cnt:
+        valid_world.parley()
+        if cnt == 0 and opt['display_examples']:
+            print(valid_world.display() + '\n~~')
+            print(valid_world.report())
+        cnt += valid_world.opt['batchsize']
```

Comment:
previously this was a call to `.get('batchsize', 1)`, right? do we need that?

Reply:
i didn't change this code, but i can do that call.

Reply:
saul goodman